parliament/parliament.js

```diff
@@ -2061,46 +1913,102 @@ app.use((req, res, next) => {
 });
 
 async function main () {
+  try {
+    await ArkimeConfig.initialize({ defaultConfigFile: `${version.config_prefix}/etc/parliament.ini` });
+
+    if (ArkimeConfig.debug === 0) {
+      ArkimeConfig.debug = parseInt(getConfig('parliament', 'debug', 0));
+    }
+    if (ArkimeConfig.debug) {
+      console.log('Debug Level', ArkimeConfig.debug);
+    }
+    internals.webBasePath = getConfig('parliament', 'webBasePath', '/');
+  } catch (err) {
+    console.log(err);
+    process.exit();
+  }
+
+  // ERROR OUT if there's no parliament config
+  if (!ArkimeConfig.getSection('parliament')) {
```

Comment:
Right now I'm erroring out if there's not parliament config created. I _could, maybe, possibly, stupidly_ use the supplied command line args (port/key/cert/config) to populate the INI. But I feel like that's overcomplicating things? But it would be nice. I'm undecided... 🤷 💭

Reply:
I think its fine to require a parliament section